app.js

```
@@ -1,77 +1,512 @@
-        taskItem.innerHTML = `
-            <div class="task-info">
-                <h3>${task.title}</h3>
-                <p>${task.description}</p>
-                <p>Priority: ${task.priority}</p>
-                <p>Due Date: ${task.dueDate}</p>
+    function render() {
+        taskList.innerHTML = '';
+
+        const normalizedSearch = state.search;
+        const visibleTasks = normalizedSearch ? filterNodes(state.tasks, normalizedSearch) : structuredClone(state.tasks);
+        visibleTasks.forEach((task) => taskList.appendChild(renderTask(task, 0)));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)